apps/ui/src/networks/offchain/api/index.ts

```diff
@@ -89,11 +89,11 @@ function getProposalState(
       quorum_type: proposal.quorumType
     });
 
-    if (
-      proposal.quorumType === 'rejection'
-        ? currentQuorum > proposal.quorum
-        : currentQuorum < proposal.quorum
-    ) {
+    if (proposal.quorumType === 'rejection') {
+      return currentQuorum > proposal.quorum ? 'rejected' : 'passed';
```

Comment:
Isn't this only checking quorum for the status, and ignoring if majority of people voted against or not ?

Reply:
Yep, if a proposal is using rejection, we need to either reject or pass